standardization/src/main/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationInterpreter.scala

```diff
@@ -18,34 +18,33 @@ package za.co.absa.enceladus.standardization.interpreter
 import org.apache.spark.sql._
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types._
-import za.co.absa.enceladus.standardization.StandardizationCommon
+import org.slf4j.{Logger, LoggerFactory}
 import za.co.absa.enceladus.standardization.interpreter.dataTypes._
 import za.co.absa.enceladus.standardization.interpreter.stages.{SchemaChecker, SparkXMLHack, TypeParser}
 import za.co.absa.enceladus.utils.error.{ErrorMessage, UDFLibrary}
-import za.co.absa.enceladus.utils.schema.SchemaUtils
 import za.co.absa.enceladus.utils.transformations.ArrayTransformations
 import za.co.absa.enceladus.utils.validation.ValidationException
 
 /**
-  * Object representing set of tools for performing the actual standardization
-  */
-object StandardizationInterpreter extends StandardizationCommon{
+ * Object representing set of tools for performing the actual standardization
+ */
+object StandardizationInterpreter{
 
-  private type ErrorCols = List[String]
+  private val logger: Logger = LoggerFactory.getLogger(this.getClass)
 
   /**
-    * Perform the standardization of the dataframe given the expected schema
-    * @param df Dataframe to be standardized
-    * @param expSchema The schema for the df to be standardized into
-    */
+   * Perform the standardization of the dataframe given the expected schema
```

Comment:
Is this autoformat? Scala style allows asterisk to be both in 2nd or 3rd column and IDEA has default to 3rd. The thing is that if we use any kind of tool in the future to generate documentation, we will need to have it set always the same. ((If the tools did not become smarter))

We should agree on a strict rule for this.

Reply:
I'd suggest we keep it consistent, througout the project we use the IntelliJ default (3rd column).